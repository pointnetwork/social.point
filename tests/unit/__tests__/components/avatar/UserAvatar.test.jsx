import React from "react";
import renderer from 'react-test-renderer';

import UserAvatar from '../../../../../src/components/avatar/UserAvatar';

describe("UserAvatar", () => {

    it('should render', () => {
        const component = renderer.create(
            <UserAvatar user={{}} address="0x12345679ABCDEF" upperLoading={false} setAlert={jest.fn()} props={{}}></UserAvatar>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});