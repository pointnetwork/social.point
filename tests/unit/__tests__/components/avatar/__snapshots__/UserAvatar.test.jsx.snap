// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAvatar should render 1`] = `
<div
  aria-label="avatar"
  className="MuiAvatar-root MuiAvatar-circular makeStyles-root-1"
  href="/profile/0x12345679ABCDEF"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#ABCDEF",
    }
  }
  to={null}
>
  <img
    alt=""
    className="MuiAvatar-img"
    src="0x0000000000000000000000000000000000000000000000000000000000000000"
  />
</div>
`;
